fitting/fitScanData.py

```diff
@@ -119,8 +125,12 @@ def fit(self, debug=False):
         random = r.TRandom3()
         random.SetSeed(0)
         for vfat in range(0,24):
-            fitTF1 = r.TF1('myERF','[3]*TMath::Erf((TMath::Max([2],x)-[0])/(TMath::Sqrt(2)*[1]))+[3]',
-                        self.calDAC2Q_m[vfat]*1+self.calDAC2Q_b[vfat],self.calDAC2Q_m[vfat]*253+self.calDAC2Q_b[vfat])
+            if self.isVFAT3:
+                fitTF1 = r.TF1('myERF','[3]*TMath::Erf((TMath::Max([2],x)-[0])/(TMath::Sqrt(2)*[1]))+[3]',
+                            self.calDAC2Q_m[vfat]*253+self.calDAC2Q_b[vfat],self.calDAC2Q_m[vfat]*1+self.calDAC2Q_b[vfat])
```

Comment:
Please forgive me my ignorance, but how the factor `253` appeared here?
and also why do we need multiplication by factor `1`: `self.calDAC2Q_m[vfat]*1`?

Reply:
We need the axis to be in charge units.  With the updated convention Henri has put forward `calDAC2Q_m` will be negative and work with the raw `CFG_CAL_DAC` register.  In voltage pulse mode low `CFG_CAL_DAC` is high charge, and high `CFG_CAL_DAC` is low charge.  The factors [253,1] then specify the charge daq range which is something like [0, 60] fC.